lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -212,6 +214,37 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     }
   end
 
+  def beacon_chain_summary(datetime) do
+    current_datetime = DateTime.utc_now()
+
+    next_datetime_summary_time =
+      datetime
+      |> BeaconChain.next_summary_date()
+
+    previous_current_date_summary_time =
```

Comment:
I think there variable name is confusing between `previous` and `next_summary_date`

Reply:
typo from the previous version